frontend/src/lib/components/Papercups.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { ChatWidget } from '@papercups-io/chat-widget'
+import { useValues } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+import { billingLogic } from 'scenes/billing/billingLogic'
+import { preflightLogic } from 'scenes/PreflightCheck/logic'
+
+export function Papercups(): JSX.Element {
+    const { user } = useValues(userLogic)
+    const { billing } = useValues(billingLogic)
+    const { realm, preflight } = useValues(preflightLogic)
+
+    return (
+        <ChatWidget
+            accountId="873f5102-d267-4b09-9de0-d6e741e0e076"
+            title="Welcome to PostHog"
+            subtitle="Ask us anything in the chat window below 😊"
+            newMessagePlaceholder="Start typing…"
+            primaryColor="#5375ff"
+            greeting="We'll respond as soon as we can. You can also check posthog.com/docs if you need assistance."
+            customer={
+                user &&
+                preflight && {
+                    email: user.email,
+                    name: user.first_name,
+                    external_id: user.distinct_id,
+                    metadata: {
+                        user_id: user.uuid,
+                        organization_name: user.organization?.name,
+                        organization_id: user.organization?.id,
+                        organization_plan: billing?.plan?.key,
+                        posthog_version: preflight.posthog_version,
+                        realm: realm,
+                        posthog_domain: location.hostname,
+                    },
+                }
+            }
+            showAgentAvailability
+        />
+    )
+}
```

Comment:
What's this? 😅

Reply:
Hm. It's an old file. This happened with the last PR I did too, not sure why. 

I'll delete the file in this commit, but not sure if it will keep happening. 

https://github.com/PostHog/posthog/pull/11302#issuecomment-1215154248